core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -474,4 +560,53 @@ private ContractMethod findMatchedInstance(List arguments) {
 
         return matchedMethod;
     }
+
+    private boolean checkParamsTypeMatched(List<Type> arguments) {
+        if(this.getInputs().size() != arguments.size()) {
+            return false;
+        }
+
+        for(int i=0; i< this.getInputs().size(); i++) {
+            ContractIOType ioType = this.getInputs().get(i);
+            if(!ioType.getType().equals(arguments.get(i).getTypeAsString())) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    private TransactionReceipt.TransactionReceiptData sendTransaction(SendOptions options, String encodedInput, TransactionReceiptProcessor processor) throws IOException, TransactionException {
+        //Make SendOptions instance by comparing with defaultSendOption and passed parameter "options"
+        //Passed parameter "options" has higher priority than "defaultSendOption" field.
+        SendOptions sendOptions = makeSendOption(options);
+        checkSendOption(sendOptions);
+
+        SmartContractExecution smartContractExecution = new SmartContractExecution.Builder()
+                .setKlaytnCall(caver.rpc.klay)
+                .setFrom(sendOptions.getFrom())
+                .setTo(this.getContractAddress())
+                .setInput(encodedInput)
+                .setGas(sendOptions.getGas())
+                .setValue(sendOptions.getValue())
+                .build();
+
+        caver.wallet.sign(sendOptions.getFrom(), smartContractExecution);
+        Bytes32 txHash = caver.rpc.klay.sendRawTransaction(smartContractExecution).send();
+
+        TransactionReceipt.TransactionReceiptData receipt = processor.waitForTransactionReceipt(txHash.getResult());
+        return receipt;
+    }
+
+    private List<Type> callFunction(ContractMethod method, String encodedInput, CallObject callObject) throws ClassNotFoundException, IOException {
+        if(callObject.getData() != null || callObject.getTo() != null) {
+            LOGGER.warn("'to' and 'data' field in CallObject will overwrite.");
+        }
+        callObject.setData(encodedInput);
+        callObject.setTo(this.getContractAddress());
```

Comment:
```suggestion
        callObject.setTo(method.getContractAddress());
```
This looks more clear maybe...?